composer.json

```diff
@@ -42,13 +42,13 @@
     "laravel/tinker": "^2.6",
     "laravel/ui": "^3.3",
     "laravelcollective/html": "^6.2",
-    "lcobucci/clock": "1.2.0",
-    "lcobucci/jwt": "^3.4.5",
+    "lcobucci/clock": "1.2.0|2.0.0",
+    "lcobucci/jwt": "^3.4.5|^4.0.4",
     "league/csv": "^9.7",
     "league/flysystem-aws-s3-v3": "^1.0",
     "league/flysystem-cached-adapter": "^1.1",
     "livewire/livewire": "^2.4",
-    "mediconesystems/livewire-datatables": "^0.4.3",
+    "mediconesystems/livewire-datatables": "^0.5.0",
```

Comment:
The lockfile takes this up to 0.5.4

Looks like no BC breaks here, and I'm honestly not sure that that we even use this - I think we've mostly stuck with our existing Bootstrap Tables implementation for most of our tabular views throughout v6

Reply:
Not directly nor indirectly used by us, as far as I can see. As we start to do more Livewire, we might though.